templates/template.rb

```diff
@@ -243,6 +253,7 @@ def locals
   "java/org/yarp/ParseResult.java",
   "java/org/yarp/AbstractNodeVisitor.java",
   "lib/yarp/node.rb",
+  "lib/yarp/sorbet.rb",
```

Comment:
I wouldn't generate this here. It's odd to have it in `lib`. I would probably make it `rbi/sorbet.rbi`.

Reply:
Totally agreed. Also, if we are generating an RBI file, which isn't the extension `.rbi`?